docs/index.rst

```diff
@@ -11,7 +11,8 @@ Blocks is developed in parallel with Fuel_, a dataset processing framework.
 
 .. warning::
    Blocks is a new project which is still under development. As such, certain
-   (all) parts of the framework are subject to change.
+   (all) parts of the framework are subject to change. The last stable (and
+   thus likely an outdated version) can be found in the "stable" branch).
```

Comment:
There's a closing parenthesis too much here, and I think the first closing parenthesis should be after "outdated", not "version".